snuba/query/snql/parser.py

```diff
@@ -118,7 +123,7 @@
     or_tuple              = space+ "OR" and_expression
 
     condition             = main_condition / parenthesized_cdn
-    main_condition        = low_pri_arithmetic space* condition_op space* (function_call / column_name / quoted_literal / numeric_literal)
+    main_condition        = low_pri_arithmetic space* condition_op space* (function_call / simple_term)
```

Comment:
Are the changes to this file related with the goal of comparing the SQL string ? If not I would split them into their own PR.

Reply:
They are related. They stemmed from adding the null_literal.